components/BaseComponents.js

```diff
@@ -24,6 +24,17 @@ export const NavButton = styled.TouchableOpacity`
   justify-content: center;
 `;
 
+export const CircleIconContainer = styled.View`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 40px;
+  width: 40px;
+  padding: 8px;
+  border-radius: 20px;
+  background-color: ${props => props.color || Colors.lighterGreen};
+`;
+
 export const ButtonContainer = styled.TouchableOpacity``;
```

Comment:
style thing? i think a lot of our other stuff has the icon vertically centered so maybe we should change this to match that, but @12aschen should chime in

Reply:
Not sure what you mean—vertically centered to what? Lines 29 & 30 should make it centered.